lib/widgets/workouts/gym_mode.dart

```diff
@@ -546,64 +555,71 @@ class _LogPageState extends State<LogPage> {
   }
 
   Widget getPlates() {
-    final plates = plateCalculator(
-      double.parse(_weightController.text),
-      BAR_WEIGHT,
-      AVAILABLE_PLATES,
-    );
-    final groupedPlates = groupPlates(plates);
-
-    return Column(
-      children: [
-        Text(
-          AppLocalizations.of(context).plateCalculator,
-          style: Theme.of(context).textTheme.titleLarge,
-        ),
-        SizedBox(
-          height: 35,
-          child: plates.isNotEmpty
-              ? Row(
-                  mainAxisAlignment: MainAxisAlignment.center,
-                  children: [
-                    ...groupedPlates.keys.map(
-                      (key) => Row(
-                        children: [
-                          Text(groupedPlates[key].toString()),
-                          const Text('×'),
-                          Container(
-                            decoration: BoxDecoration(
-                              color: Theme.of(context).colorScheme.primaryContainer,
-                              shape: BoxShape.circle,
-                            ),
-                            child: Padding(
-                              padding: const EdgeInsets.symmetric(horizontal: 3),
-                              child: SizedBox(
-                                height: 35,
-                                width: 35,
-                                child: Align(
-                                  alignment: Alignment.center,
-                                  child: Text(
-                                    key.toString(),
-                                    style: const TextStyle(
-                                      fontWeight: FontWeight.bold,
+    final plates = plateCalculator(num.parse(_weightController.text),BAR_WEIGHT,AVAILABLE_PLATES);
+    Map<num,int> groupedPlates;
+    groupedPlates = groupPlates(plates);
+    return Consumer<PlateWeights>(builder: (context,plateProvider, child) =>
+      SingleChildScrollView(
+        child: Column(
+          children: [
+            Row(
+              mainAxisAlignment: MainAxisAlignment.center,
+                children: [
+                  ElevatedButton(onPressed: () {
+                      plateProvider.flag=false;
+                      plateProvider.setWeight(num.parse(_weightController.text));
+                      Navigator.of(context).push(MaterialPageRoute(builder: (context)=>const AddPlateWeights()));
+                    },
+                    child: const Text("Enter custom Denomination's")
```

Comment:
I would make this a bit less prominent, a gear icon left to "Plates" would probably be enough